src/openrct2/ride/Vehicle.cpp

```diff
@@ -6262,7 +6262,7 @@ int32_t Vehicle::UpdateMotionDodgems()
         }
     }
 
-    auto collideSprite = EntityId::GetNull();
+    std::optional<EntityId> collideSprite = EntityId::GetNull();
```

Comment:
The value will be overwritten, just leave it valueless.

Reply:
You are correct, done.